ddp-workspace/projects/ddp-cgc/src/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+
+    <base href="/" />
+
+    <link rel="icon" type="image/x-icon" href="favicon.ico" />
```

Comment:
Looks like the `favicon.ico` file right now is the Angular icon. I'm assuming that's temporary?

Reply:
Yes, it's temporary.